src/cc/libbpf.c

```diff
@@ -1203,6 +1203,23 @@ int bpf_attach_kfunc(int prog_fd)
   return ret;
 }
 
+int bpf_detach_lsm(int prog_fd, char *func)
```

Comment:
could you remove bpf_detach_lsm? There is no need from python side as python code will close the fd. We can add it later if it is truely used in C++ side.

Reply:
@yonghong-song Yes I can remove. But I was copying what was done for kfunc. Perhaps it should be removed for kfunc as well?

Reply:
Yes, let us remove it as well. The functions are not used. Let us add them if there is a need in C++ side later. Thanks!

Reply:
@yonghong-song I have now removed both detach_lsm and detach_kfunc